frontend/src/metabase/parameters/components/ParametersList/types.ts

```diff
@@ -12,7 +12,7 @@ export type ParametersListProps = {
   {
     className: string;
 
-    question: Question;
+    question: Question | null;
```

Comment:
Twas a missing typescript error

Reply:
I log the actual result from the public dashboard and it's actually `undefined`. That makes sense since we won't likely pass `question={null}` for public/embedded dashboards. Can you double-check that this type is correct?

We also need to fix the `dashboard` type to be the same on the line below. 

![image](https://github.com/metabase/metabase/assets/1937582/39f543ba-7a1f-4317-9a29-a2c8a5860875)

Reply:
Will take a look

Reply:
Yep, I went through the instances of the `SyncedParametersList` and `ParametersList` and it looks like they both are `[Question/Dashboard] | undefined`. I'll change that now